### ISSUE TITLE
Compiling failing on ZNC-1.0

### ISSUE THREAD

**Issue by unlobito** [wired/colloquypush#32]
Attempting to compile the latest version of colloquypush on ZNC-1.0 results in a fatal error.

```
$ curl -LO http://github.com/wired/colloquypush/raw/master/znc/colloquy.cpp
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0   178    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   131  100   131    0     0    128      0  0:00:01  0:00:01 --:--:--   909
100 26711  100 26711    0     0  13664      0  0:00:01  0:00:01 --:--:--  207k

$ znc-buildmod colloquy.cpp
Building "colloquy.so" for ZNC 1.0... colloquy.cpp:9:17: fatal error: znc.h: No such file or directory
compilation terminated.
[ !! ] Error while building "colloquy.so"
```

**Comment by wired** [comment]
The module needs an update for znc-1.0.
I plan to update it the following days :)

**Comment by an0key** [comment]
Did you get anywhere with this?

**Comment by unlobito** [comment]
This was fixed in 6aca83bfaf9c7d01f939ba2898d34c9c485aee22.
